doc/sphinxman/source/scf.rst

```diff
@@ -782,7 +782,7 @@ a cutoff for the value of basis functions at grid points. This keyword is
 used to determine the radial extent of the each basis shell, and it is the
 COSX analogue to |scf__dft_basis_tolerance|.
 
-The |scf__cosx_incfock| keyword (defaults to ``false``) increases performance
+The |scf__incfock| keyword (defaults to ``true``) increases performance
```

Comment:
Now that scf_cosx_incfock is being removed as a keyword, I wonder how much sense it makes to keep a full paragraph of discussion surrounding scf_incfock in the COSX section. Maybe a small comment about scf_incfock can be included in the previous paragraph instead?

Also, assuming we do keep this paragraph of discussion, we would need to remove the final sentence of it entirely, as this PR switches IncFock to be true by default for COSX.

Reply:
Agreed. There should be a paragraph of `incfock` discussion in general, mentioning that this is only allowed for algorithms X, Y, Z. Unless specific algorithms have special `incfock` options, no need to discuss in general.

Reply:
In the `incfock` section, I explained which algorithms would have `incfock` on by default